Formula/xidel.rb

```diff
@@ -0,0 +1,77 @@
+class Xidel < Formula
+  desc "XPath/XQuery 3.0, JSONiq interpreter to extract data from HTML/XML/JSON"
+  homepage "https://www.videlibri.de/xidel.html"
+  license "GPL-3.0-or-later"
+
+  class TargetedResource < Resource
+    def initialize(*rest, **kw_rest)
+      @target = nil
+      super(*rest, **kw_rest)
+    end
+
+    def target(val = nil)
+      return @target if val.nil?
+
+      @target = val
+    end
+  end
+
+  stable do
+    url "https://github.com/benibela/xidel/releases/download/Xidel_0.9.8/xidel-0.9.8.src.tar.gz"
+    sha256 "72b5b1a2fc44a0a61831e268c45bc6a6c28e3533b5445151bfbdeaf1562af39c"
+  end
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+    regex(%r{href=.*?/tag/Xidel_(\d+(?:\.\d+)+)["' >]}i)
+  end
+
+  head do
+    url "https://github.com/benibela/xidel.git", branch: "master"
+
+    resource "flre", TargetedResource do
+      url "https://github.com/benibela/flre.git", branch: "master"
+      target "import/#{name}"
+    end
+    resource "internettools", TargetedResource do
+      url "https://github.com/benibela/internettools.git", branch: "master"
+      target "import/#{name}"
+    end
+    resource "pasdblstrutils", TargetedResource do
+      url "https://github.com/BeRo1985/pasdblstrutils.git", branch: "master"
+      target "import/#{name}"
+    end
+    resource "rcmdline", TargetedResource do
+      url "https://github.com/benibela/rcmdline.git", branch: "master"
+      target "import/#{name}"
+    end
+    resource "synapse", TargetedResource do
+      url "http://svn.code.sf.net/p/synalist/code/synapse/40"
+      target "import/#{name}"
+    end
+  end
```

Comment:
Having `head` looks too complicated. Let's get rid of it.
```suggestion
```

Reply:
I'm currently running HEAD; 0.98.0 was released 2018-04-02 and the latest macOS [binary of a prerelease](https://sourceforge.net/projects/videlibri/files/Xidel/Xidel%20development/) was uploaded 2019-03-10. Until Xidel starts releasing regularly again, having HEAD in the formula is very useful.

Reply:
It might be useful, but it's hard to maintain. You're better off asking them to start releasing again

Reply:
Would you prefer I did some bodge that avoided the `resource` system involving scraping the `.travis.yml` file in the repository instead?

https://github.com/benibela/xidel/blob/a933a995e523d9c9076dd969e3a006aa0c6e7e40/.travis.yml#L18-L22

It's 5 dependencies, 4 of which have stayed the same for years. This isn't really a moving target.